docker/ssl/authorization.js

```diff
@@ -138,18 +138,6 @@ function setKey(r) {
         return '';
     }
 
-    if (r.uri.startsWith('/ws') && (!auth || !auth.length)) {
-        var a;
-        for (a in r.args) {
-            if (a == 'authorization' && r.args[a] === clientKey) {
-                return clientKey;
-            }
-        }
-
-        r.error('Authorization param does not match certificate');
-        return '';
-    }
-
```

Comment:
Please don't remove this. We want to keep MQTT over WS.

Reply:
ok!